torch/lib/c10d/TCPStore.cpp

```diff
@@ -204,6 +273,15 @@ void TCPStoreDaemon::compareSetHandler(int socket) {
   } else {
     if (pos->second == currentValue) {
       pos->second = std::move(newValue);
+
```

Comment:
Don't we need to handle `ADD` and `DELETE` as well?

Reply:
Good point, updated the PR